modules/gitcoinPassportEligibility.json

```diff
@@ -0,0 +1,252 @@
+{
+  "name": "Gitcoin Passport Eligibility",
+  "details": [
+    "A Hats Protocol eligibility module that checks if an address has a minimum score that passes the GitcoinPassportDecoder or custom threshold."
+  ],
+  "links": [
+    {
+      "label": "GitHub",
+      "link": "https://github.com/daocoa/gitcoin-passport-eligibility/"
+    }
+  ],
+  "parameters": [
+    {
+      "label": "Gitcoin Passport Decoder",
+      "functionName": "gitcoinPassportDecoder",
+      "displayType": "default",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as an address in the module smart contract. Is this correct or should I use `token`?"
+
+    },
+    {
+      "label": "Score Criterion",
+      "functionName": "scoreCriterion",
+      "displayType": "default",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as a uint256 in the module smart contract. Is this correct or should I use `amountWithDecimals` or `hat`?"
+    }
+  ],
+  "type": {
+    "eligibility": true,
+    "toggle": false,
+    "hatter": false
+  },
+  "tags": [],
+  "implementationAddress": "0x4Ce2132f7cC5d5625D757CeDBE67D26CB4C0ec0f",
+  "deployments": [
+    {
+      "chainId": "10",
+      "block": "TBA"
+    },
+    {
+      "chainId": "11155420",
+      "block": "11734099"
+    },
+    {
+      "chainId": "11155111",
+      "block": "5868541"
+    }
+  ],
+  "creationArgs": {
+    "useHatId": true,
+    "immutable": [
+      {
+        "name": "Gitcoin Passport Decoder",
+        "description": "Decoder address",
+        "type": "address",
+        "example": "0xe53C60F8069C2f0c3a84F9B3DB5cf56f3100ba56",
+        "displayType": "address",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as an address in the module smart contract. Is this correct or should I use `token`?"
```

Comment:
`default` will let the Hats app know to display this field according to its solidity type (`address`) and so the app will display that as a regular contract